From fe236dbdd77e0d692927b5f53db1e3154502e59e Mon Sep 17 00:00:00 2001
From: Sanjay Singh Chauhan <schauhan@nvidia.com>
Date: Thu, 7 Feb 2013 12:44:06 +0530
Subject: [PATCH 05/10] power: Add POWER_SHIELD_STREAMING to power hints

Need POWER_HINT_SHIELD_STREAMING to boost CPU for game streaming
and avoid jitter in video decode and Depacketizer time
due to higher thread scheduling time.

Bug 1196851

Reviewed-on: http://git-master/r/198279
(cherry picked from commit 0e23206a7cde3dde7c1c3015041c1d5f3ce3f7e1)

Change-Id: Ib64faebf6b496b3efdb11c7a61620f0c5657e830
Reviewed-on: http://git-master/r/203508
Reviewed-by: Automatic_Commit_Validation_User
GVS: Gerrit_Virtual_Submit
Reviewed-by: Mitch Luban <mluban@nvidia.com>
Tested-by: Mitch Luban <mluban@nvidia.com>
Reviewed-by: Bharat Nihalani <bnihalani@nvidia.com>
---
 include/hardware/power.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/hardware/power.h b/include/hardware/power.h
index b343fe3..cc191bc 100644
--- a/include/hardware/power.h
+++ b/include/hardware/power.h
@@ -43,7 +43,8 @@ typedef enum {
     POWER_HINT_INTERACTION = 0x00000002,
     POWER_HINT_APP_PROFILE = 0x00000003,
     POWER_HINT_APP_LAUNCH = 0x00000004,
-    POWER_HINT_COUNT = 0x00000005,
+    POWER_HINT_SHIELD_STREAMING = 0x00000005,
+    POWER_HINT_COUNT = 0x00000006,
 } power_hint_t;
 
 /*
-- 
1.8.1.5

